pkg/scheduler/scheduler.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import "github.com/dapr/kit/logger"
+
+var log = logger.NewLogger("dapr.runtime.scheduler")
+
+type SidecarConnDetails struct {
+	Namespace string
```

Comment:
Exported?

Reply:
Yes, I am open to moving it if you have a suggestion. Rn its used in:

- pkg/runtime/scheduler/scheduler.go in the scheduler manager pkg for sending the sidecar ns/appID along on the WatchJob stream req
- pkg/scheduler/server/internal/pool.go for the same reason about but from the scheduler perspective. its for the conn pool to send the trigger back to the right namespace appID. 
- pkg/scheduler/server/api.go in the watchJob bc I send those details along to the connPool pkg